Framework/ICat/src/CatalogDownloadDataFiles.cpp

```diff
@@ -169,6 +169,8 @@ std::string CatalogDownloadDataFiles::doDownloadandSavetoLocalDrive(
     Poco::Net::SecureStreamSocket *socket =
         new Poco::Net::SecureStreamSocket(context);
     Poco::Net::HTTPSClientSession session(*socket);
```

Comment:
How about we take this change out and create a separate issue? It feels too big/interesting to hide in the middle of a code clean-up.